pkg/e2e/e2e.go

```diff
@@ -241,8 +233,8 @@ func runGinkgoTests() error {
 	gomega.RegisterFailHandler(ginkgo.Fail)
 
 	ginkgoConfig.DefaultReporterConfig.NoisySkippings = !viper.GetBool(config.Tests.SuppressSkipNotifications)
-	ginkgoConfig.GinkgoConfig.SkipString = viper.GetString(config.Tests.GinkgoSkip)
-	ginkgoConfig.GinkgoConfig.FocusString = viper.GetString(config.Tests.GinkgoFocus)
+	ginkgoConfig.GinkgoConfig.SkipStrings = []string{viper.GetString(config.Tests.GinkgoSkip)}
```

Comment:
Is this an unrelated bugfix?

Reply:
This was related to upgrading Ginkgo versions and will be removed.